tests/weight_loading/models.txt

```diff
@@ -0,0 +1,6 @@
+gptq_marlin, robertgshaw2/zephyr-7b-beta-channelwise-gptq, main
+gptq_marlin, TheBloke/Llama-2-7B-GPTQ, main
```

Comment:
Trying to simplify this config, I think it would be better if it was just a list of model stubs

- you dont need the `gptq_marlin` in your test, vLLM will automatically detect
- we can just pick models which don't have to worry about revision

Reply:
@robertgshaw2-neuralmagic 

2 things here:
1. I think we want test cases around running specific configs. This lets us do that. I understand it should be able to autodetect. But the goal is the detect the weight loading for a specific quantization config 
2. Sometimes the quantization used be autodetected based on compatibility and its not what we want to test (example: awq models running on awq-marlin)